src/aspire/source/coordinates.py

```diff
@@ -569,9 +544,7 @@ def __init__(
         :param max_rows: Maximum number of particles to read. (If `None`, will attempt to load all particles)
         :param B: CTF envelope decay factor
         :param symmetry_group: A `SymmetryGroup` object or string corresponding to the symmetry of the molecule.
-        :param pixel_size: Pixel size of the images in angstroms.
-            Default `None` will attempt to infer `pixel_size` from
-            `CTFFilter` objects when available.
+        :param pixel_size: Pixel size of the images in angstroms (Required).
```

Comment:
Same. I suggest we move this up to just after files and remove the default.

Reply:
Sure. Done.